apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/readme.md

```diff
@@ -33,6 +33,7 @@ When using default options, logs will appear under these filter settings in the
 * **IncludeCallSiteStackTrace** - Include LogEntrySourceLocation extracted from NLog callsite (Also include filename and linenumber)
 * **IncludeEventProperties** - Includes structured logging properties from NLog LogEventInfo.Properties
 * **IncludeMdlc** - Include async thread context properties from NLog MappedDiagnosticsLogicalContext
+* **SendJsonPayload** - Instead of sending properties as custom labels, then they are sent as Json-Properties

```

Comment:
Please also state that the log message will be in `JsonPayload`, not `TextPayload`

Reply:
Fixed